io/disk/sysfs_att.py.data/sysfs_att.yaml

```diff
@@ -0,0 +1,14 @@
+disk: "sdd"
+queue_attributes: !mux
+    io_poll:
+        attribute_name: "io_poll"
+        values: "0 1"
+    io_poll_delay:
+        attribute_name: "io_poll_delay"
+        values: "-1 0 1"
+    iostats:
+        attribute_name: "iostats"
+        values: "0 1"
+    io_timeout:
+        attribute_name: "io_timeout"
+        values: "1 30000"
```

Comment:
How did you arrive at these values? On one system I can see a value of 120000.

Reply:
I choose 30000 as a random value. Yes, we can use values upto 120000